### DISCUSSION TITLE
Has the ‘Change Scene’ feature disappeared?

### DISCUSSION THREAD

**Discussion by PoliyasRoden** [dialogic-godot/dialogic#2097] [Q&A] [answered]
When dialogic was in the days of godot3.X, there was a 'change scene'.

It doesn't appear in godot4.X.

Am I doing something wrong?

If not, are there any further plans?

**Comment by metanoder** [comment, marked as answer]
[Change Scene is deprecated](https://discord.com/channels/628713677239091231/1089341091729395762/1151530895426195686).

Use Signals or autoload func call to change scene. 

via Autoload
```
func change_scene(argument:String):
        get_tree().change_scene_to_file(argument)
```

Call Autoload on timeline ie...
`do autoload.change_scene("res://scene_name.tscn")`

[Connected signal ie...](https://dialogic-docs.coppolaemilio.com/dialogic-signals.html)
_To react to the signal event, connect to it before starting your dialog:_
```
func _ready():

     Dialogic.signal_event.connect(_on_dialogic_signal)
     Dialogic.start("My_Dialog_Timeline")

func _on_dialogic_signal(argument:String):
        get_tree().change_scene_to_file(argument)

func _on_dialogic_signal(argument:Dictionary):
    if argument is Dictionary:
        if arg["method"] == "change_scene": # checks method
			get_tree().change_scene_to_file(argument["path"])
```
call via string...
`[signal arg="res://scene_name.tscn"]`

call via dictionary...
`[signal arg_type="dict" arg="{"method":"change_scene","path":"res://scene_name.tscn"}"]`
